serverless.yaml

```diff
@@ -110,6 +111,7 @@ functions:
     provisionedConcurrency: 5
     environment:
       EXPORT_STATE_MACHINE_ARN: !Ref BulkExportStateMachine
+      ALLOW_USER_SCOPE_FOR_SYSTEM_EXPORT: !Ref AllowUserScopeForSystemExport

```

Comment:
I don't think this parameter will be used so another option is this to be configured via the config.ts file. May want to solicit opinions around the team.

Reply:
I like it to be configured via config.js file. Will close out this PR.